Utility/PiecesTemplates.FCMacro

```diff
@@ -0,0 +1,421 @@
+
+    def mostrarComboGroups(self):
+        self.form.comboEliminarGrupo.clear()
+        for cont in range(self.longitudListaGrupos):
+            self.form.comboEliminarGrupo.addItem(listGroups[cont])
+
+    def initStrings(self):
+        tr = self.translate
+        if g_num_lang != 1:
+            title = "(" + strLang[g_num_lang][0] + ")"
+        else:
+            title = ""
+        self.form.setWindowTitle(__Name__ + " " + title)
+        self.form.grupoPrincipal.setTitle(tr("Creación de Piezas"))
+        self.form.etNombrePieza.setText(tr("Nombre de la Pieza:"))
+        self.form.botonAyuda.setToolTip(tr("Información de Uso"))
+        self.form.edicionNombrePieza.setText(tr(""))
+        self.form.etGrupo.setText(tr("Grupos/Directorios a crear:"))            
+        self.form.botonCrear.setText(tr("Crear Estructura de Pieza"))
+        self.form.etVersion.setText(tr("Versión:"))
+        self.form.version.setText(__Version__)
+        self.form.tipoVersion.setText(__Status__)
+        self.form.infoPiezasCreadas.setText("")
+        self.form.etPiezasCreadas.setText(tr("Listado Piezas Creadas:"))
+        self.form.numPiezasCreadas.setText(tr("Nº de Piezas Creadas:"))
+        self.form.grupoOperaciones.setTitle(tr("Operaciones sobre Grupos"))
+        self.form.anadirGrupo.setText(tr("Añadir Grupo"))
+        self.posGroup()
+        self.form.eliminarGrupo.setText(tr("Eliminar Grupo"))
+        self.form.etPiezasCreadas.setEnabled(False)
+        self.form.numPiezas.setEnabled(False)
+
+    def mostrarHelp(self):
+        tr = self.translate
+        titleMH = tr("Información") + " (" + __Name__ + ")"
+        stringMH = tr("Este script (macro) genera una estructura de grupos (directorios) en una pieza, de manera que sea más fácil organizar las distintas operaciones,referencias y objetos de esa pieza.")
+        stringMH += cIntro + cIntro + tr("Los grupos inicialmente están predefinidos, pero pueden ser cambiados fácilmente.")
+        reply = QtGui.QMessageBox.information(None,titleMH,stringMH)
+
+    def posGroup(self):
+        tr = self.translate
+        self.form.comboPosicionGrupo.clear()
+
+        if (self.longitudListaGrupos == 0):
+            self.form.comboPosicionGrupo.addItem(tr("el primero"))
+            self.form.eliminarGrupo.setEnabled(False)
+        else:
+            max = self.longitudListaGrupos
+            for cont in range(max):
+                if cont == 0:
+                    self.form.comboPosicionGrupo.addItem(tr("el primero"))
+                if (cont !=0) and (cont < max):
+                    self.form.comboPosicionGrupo.addItem(str(cont+1))
+                if (cont == max-1):
+                    self.form.comboPosicionGrupo.addItem(tr("el último"))                    
+
+    def nombrePiezaCambia(self):
+        if (self.form.edicionNombrePieza.text() != ''):
+            self.form.botonCrear.setEnabled(True)
+        else:
+            self.form.botonCrear.setEnabled(False)
```

Comment:
Maybe as one line: `self.form.botonCrear.setEnabled(not bool(self.form.edicionNombrePieza.text())`?